compiler-test/src/commonTest/kotlin/com/kitakkun/backintime/test/specific/MutableCollectionsCaptureTest.kt

```diff
@@ -72,6 +72,7 @@ class MutableCollectionsCaptureTest : BackInTimeDebugServiceTest() {
         holder.mutableMapTest()
         delay(100)
 
+        println(notifyValueChangeEvents.map { it.value })
```

Comment:
code for debugging. can be removed.

Reply:
deleted on [Remove code for debugging](https://github.com/kitakkun/back-in-time-plugin/pull/138/commits/e1b30865e437436389f7d10fa77591e778c6ec4d)